apps/etterlatte-saksbehandling-ui/client/src/shared/types/IDetaljertBehandling.ts

```diff
@@ -56,14 +56,14 @@ export enum IBehandlingStatus {
   AVBRUTT = 'AVBRUTT',
 }
 
-export enum IUtenlandstilsnittType {
+export enum INasjonalitetsType {
   NASJONAL = 'NASJONAL',
   UTLANDSTILSNITT = 'UTLANDSTILSNITT',
   BOSATT_UTLAND = 'BOSATT_UTLAND',
 }
 
 export interface IUtenlandstilsnitt {
-  type: IUtenlandstilsnittType
+  type: INasjonalitetsType
```

Comment:
Veldig nitpick, men når man først endrer navnet: Jeg personlig synes man _enten_ må velge camelCase _uten norsk "fuge-s", dvs her `INasjonalitetType`, eller rett-frem norsk, dvs `INasjonalitetstype`.

Reply:
Enig i det forsåvidt, var en lazy copy fra et gammelt objekt som var ment for samme funksjonalitet som utlandstilsnitt som jeg synes ga mer verdi her.

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2582/commits/2a18d588fdf18cea20119e6f67a7493cea05e244